test/driver_test.rb

```diff
@@ -51,4 +51,29 @@ class DriverTestClass < Object
       DriverTestClass.new.schema_path
     end
   end
+
+  test '#schema_hash with json format argument returns correct hash data' do
+    Rails.application.config.schema_conformist.schema_path = Rails.root.join('public', 'swagger.json')
+    json_data = <<-JSON.strip_heredoc
```

Comment:
I thought available ruby version for schema_conformist is 2.2 or above.  
Because ruby 2.2 doesn't work `<<~`, I use `strip_heredoc` instead of `<<~`.  
But maintenance of ruby 2.2 will be ended since the end of the March of 2018.  
So I fix from `strip_heredoc` to `<<~`. 

https://github.com/kymmt90/schema_conformist/pull/5/files#diff-b814a1fc6d7f1b214dc94e1bc1169d66

Reply:
👍 

I think this is only test so it seems that there is no need to be sensitive to Ruby versions 😎